cmd/cwa/testdata/test.rs

```diff
@@ -0,0 +1,97 @@
+extern crate core;
```

Comment:
Unnecessary

Reply:
Nope, it's actually needed.

```
λ ./build.sh 
+ rustc --target wasm32-unknown-unknown -o test.wasm test.rs
error[E0433]: failed to resolve. Use of undeclared type or module `core`
  --> test.rs:62:37
   |
62 |         let arg_str: &str = unsafe{ core::str::from_utf8_unchecked(&arg_val[..res as usize]) };
   |                                     ^^^^ Use of undeclared type or module `core`

error[E0433]: failed to resolve. Use of undeclared type or module `core`
  --> test.rs:81:19
   |
81 |         val_str = core::str::from_utf8_unchecked(&envvar_val[..res as usize]);
   |                   ^^^^ Use of undeclared type or module `core`

error: aborting due to 2 previous errors
```

Reply:
That's because you were still referencing `core::str` instead of `std::str`, which you swapped to elsewhere.